diff --git a/Assets/Animations/OrcRider/Attack.anim b/Assets/Animations/OrcRider/Attack.anim
index d0a94bf3..64f08222 100644
--- a/Assets/Animations/OrcRider/Attack.anim
+++ b/Assets/Animations/OrcRider/Attack.anim
@@ -21,25 +21,23 @@ AnimationClip:
   - serializedVersion: 2
     curve:
     - time: 0
-      value: {fileID: -3319210219437721504, guid: 78f2309cbc878bd4c93862b47ae40ebf, type: 3}
-    - time: 0.099999905
-      value: {fileID: -7450898028205555343, guid: 78f2309cbc878bd4c93862b47ae40ebf, type: 3}
-    - time: 0.20000005
-      value: {fileID: 1078851311737526472, guid: 78f2309cbc878bd4c93862b47ae40ebf, type: 3}
-    - time: 0.29999995
-      value: {fileID: -3386564002228378813, guid: 78f2309cbc878bd4c93862b47ae40ebf, type: 3}
-    - time: 0.39999986
-      value: {fileID: 7093141418422216890, guid: 78f2309cbc878bd4c93862b47ae40ebf, type: 3}
+      value: {fileID: -4805291195941671494, guid: 0c064d11e4b114d46a1b86fe764bb87a, type: 3}
+    - time: 0.12
+      value: {fileID: -898491107783281163, guid: 0c064d11e4b114d46a1b86fe764bb87a, type: 3}
+    - time: 0.25
+      value: {fileID: 2851206345125205499, guid: 0c064d11e4b114d46a1b86fe764bb87a, type: 3}
+    - time: 0.38
+      value: {fileID: 5773026405177017699, guid: 0c064d11e4b114d46a1b86fe764bb87a, type: 3}
     - time: 0.5
-      value: {fileID: 3815532623530034742, guid: 78f2309cbc878bd4c93862b47ae40ebf, type: 3}
-    - time: 0.5999999
-      value: {fileID: 4255114554553264937, guid: 78f2309cbc878bd4c93862b47ae40ebf, type: 3}
-    - time: 0.70000005
-      value: {fileID: 2398725940308958766, guid: 78f2309cbc878bd4c93862b47ae40ebf, type: 3}
-    - time: 0.79999995
-      value: {fileID: 4593478631824171390, guid: 78f2309cbc878bd4c93862b47ae40ebf, type: 3}
-    - time: 0.89
-      value: {fileID: 4593478631824171390, guid: 78f2309cbc878bd4c93862b47ae40ebf, type: 3}
+      value: {fileID: -3239606791055260546, guid: 0c064d11e4b114d46a1b86fe764bb87a, type: 3}
+    - time: 0.62
+      value: {fileID: 5051900723028539795, guid: 0c064d11e4b114d46a1b86fe764bb87a, type: 3}
+    - time: 0.75
+      value: {fileID: 837233935513997722, guid: 0c064d11e4b114d46a1b86fe764bb87a, type: 3}
+    - time: 0.88
+      value: {fileID: 7414343374642175032, guid: 0c064d11e4b114d46a1b86fe764bb87a, type: 3}
+    - time: 1
+      value: {fileID: 6936758103650135619, guid: 0c064d11e4b114d46a1b86fe764bb87a, type: 3}
     attribute: m_Sprite
     path: 
     classID: 212
@@ -51,19 +49,37 @@ AnimationClip:
     m_Center: {x: 0, y: 0, z: 0}
     m_Extent: {x: 0, y: 0, z: 0}
   m_ClipBindingConstant:
-    genericBindings: []
-    pptrCurveMapping: []
+    genericBindings:
+    - serializedVersion: 2
+      path: 0
+      attribute: 0
+      script: {fileID: 0}
+      typeID: 212
+      customType: 23
+      isPPtrCurve: 1
+      isIntCurve: 0
+      isSerializeReferenceCurve: 0
+    pptrCurveMapping:
+    - {fileID: -4805291195941671494, guid: 0c064d11e4b114d46a1b86fe764bb87a, type: 3}
+    - {fileID: -898491107783281163, guid: 0c064d11e4b114d46a1b86fe764bb87a, type: 3}
+    - {fileID: 2851206345125205499, guid: 0c064d11e4b114d46a1b86fe764bb87a, type: 3}
+    - {fileID: 5773026405177017699, guid: 0c064d11e4b114d46a1b86fe764bb87a, type: 3}
+    - {fileID: -3239606791055260546, guid: 0c064d11e4b114d46a1b86fe764bb87a, type: 3}
+    - {fileID: 5051900723028539795, guid: 0c064d11e4b114d46a1b86fe764bb87a, type: 3}
+    - {fileID: 837233935513997722, guid: 0c064d11e4b114d46a1b86fe764bb87a, type: 3}
+    - {fileID: 7414343374642175032, guid: 0c064d11e4b114d46a1b86fe764bb87a, type: 3}
+    - {fileID: 6936758103650135619, guid: 0c064d11e4b114d46a1b86fe764bb87a, type: 3}
   m_AnimationClipSettings:
     serializedVersion: 2
     m_AdditiveReferencePoseClip: {fileID: 0}
     m_AdditiveReferencePoseTime: 0
     m_StartTime: 0
-    m_StopTime: 0.9
+    m_StopTime: 1.01
     m_OrientationOffsetY: 0
     m_Level: 0
     m_CycleOffset: 0
     m_HasAdditiveReferencePose: 0
-    m_LoopTime: 0
+    m_LoopTime: 1
     m_LoopBlend: 0
     m_LoopBlendOrientation: 0
     m_LoopBlendPositionY: 0
diff --git a/Assets/Animations/OrcRider/Charge.anim b/Assets/Animations/OrcRider/Charge.anim
index 60f5fda5..71b1d59a 100644
--- a/Assets/Animations/OrcRider/Charge.anim
+++ b/Assets/Animations/OrcRider/Charge.anim
@@ -21,23 +21,27 @@ AnimationClip:
   - serializedVersion: 2
     curve:
     - time: 0
-      value: {fileID: -950124526254257863, guid: 78f2309cbc878bd4c93862b47ae40ebf, type: 3}
-    - time: 0.100000024
-      value: {fileID: -141208729075187540, guid: 78f2309cbc878bd4c93862b47ae40ebf, type: 3}
-    - time: 0.20000005
-      value: {fileID: -5604554462252794442, guid: 78f2309cbc878bd4c93862b47ae40ebf, type: 3}
-    - time: 0.30000007
-      value: {fileID: -3234594974429336707, guid: 78f2309cbc878bd4c93862b47ae40ebf, type: 3}
-    - time: 0.39999998
-      value: {fileID: -3525687898906636510, guid: 78f2309cbc878bd4c93862b47ae40ebf, type: 3}
-    - time: 0.5
-      value: {fileID: -2705282085282299448, guid: 78f2309cbc878bd4c93862b47ae40ebf, type: 3}
+      value: {fileID: -7304371395691662354, guid: 6bd053b628698c54695c3b11872171e4, type: 3}
+    - time: 0.12
+      value: {fileID: -5502613770763856643, guid: 6bd053b628698c54695c3b11872171e4, type: 3}
+    - time: 0.24
+      value: {fileID: -5171600912216366864, guid: 6bd053b628698c54695c3b11872171e4, type: 3}
+    - time: 0.36
+      value: {fileID: -7304371395691662354, guid: 6bd053b628698c54695c3b11872171e4, type: 3}
+    - time: 0.48
+      value: {fileID: -5502613770763856643, guid: 6bd053b628698c54695c3b11872171e4, type: 3}
     - time: 0.6
-      value: {fileID: 6980226851984165872, guid: 78f2309cbc878bd4c93862b47ae40ebf, type: 3}
-    - time: 0.6999999
-      value: {fileID: 4538685612813587589, guid: 78f2309cbc878bd4c93862b47ae40ebf, type: 3}
-    - time: 0.78999996
-      value: {fileID: 4538685612813587589, guid: 78f2309cbc878bd4c93862b47ae40ebf, type: 3}
+      value: {fileID: -5171600912216366864, guid: 6bd053b628698c54695c3b11872171e4, type: 3}
+    - time: 0.72
+      value: {fileID: 4071341206943624054, guid: 6bd053b628698c54695c3b11872171e4, type: 3}
+    - time: 0.84
+      value: {fileID: 1562773691389614494, guid: 6bd053b628698c54695c3b11872171e4, type: 3}
+    - time: 0.96
+      value: {fileID: -3231506305399026275, guid: 6bd053b628698c54695c3b11872171e4, type: 3}
+    - time: 1.08
+      value: {fileID: 3518738520941088448, guid: 6bd053b628698c54695c3b11872171e4, type: 3}
+    - time: 1.2
+      value: {fileID: 3990685421678634285, guid: 6bd053b628698c54695c3b11872171e4, type: 3}
     attribute: m_Sprite
     path: 
     classID: 212
@@ -49,19 +53,39 @@ AnimationClip:
     m_Center: {x: 0, y: 0, z: 0}
     m_Extent: {x: 0, y: 0, z: 0}
   m_ClipBindingConstant:
-    genericBindings: []
-    pptrCurveMapping: []
+    genericBindings:
+    - serializedVersion: 2
+      path: 0
+      attribute: 0
+      script: {fileID: 0}
+      typeID: 212
+      customType: 23
+      isPPtrCurve: 1
+      isIntCurve: 0
+      isSerializeReferenceCurve: 0
+    pptrCurveMapping:
+    - {fileID: -7304371395691662354, guid: 6bd053b628698c54695c3b11872171e4, type: 3}
+    - {fileID: -5502613770763856643, guid: 6bd053b628698c54695c3b11872171e4, type: 3}
+    - {fileID: -5171600912216366864, guid: 6bd053b628698c54695c3b11872171e4, type: 3}
+    - {fileID: -7304371395691662354, guid: 6bd053b628698c54695c3b11872171e4, type: 3}
+    - {fileID: -5502613770763856643, guid: 6bd053b628698c54695c3b11872171e4, type: 3}
+    - {fileID: -5171600912216366864, guid: 6bd053b628698c54695c3b11872171e4, type: 3}
+    - {fileID: 4071341206943624054, guid: 6bd053b628698c54695c3b11872171e4, type: 3}
+    - {fileID: 1562773691389614494, guid: 6bd053b628698c54695c3b11872171e4, type: 3}
+    - {fileID: -3231506305399026275, guid: 6bd053b628698c54695c3b11872171e4, type: 3}
+    - {fileID: 3518738520941088448, guid: 6bd053b628698c54695c3b11872171e4, type: 3}
+    - {fileID: 3990685421678634285, guid: 6bd053b628698c54695c3b11872171e4, type: 3}
   m_AnimationClipSettings:
     serializedVersion: 2
     m_AdditiveReferencePoseClip: {fileID: 0}
     m_AdditiveReferencePoseTime: 0
     m_StartTime: 0
-    m_StopTime: 0.79999995
+    m_StopTime: 1.21
     m_OrientationOffsetY: 0
     m_Level: 0
     m_CycleOffset: 0
     m_HasAdditiveReferencePose: 0
-    m_LoopTime: 0
+    m_LoopTime: 1
     m_LoopBlend: 0
     m_LoopBlendOrientation: 0
     m_LoopBlendPositionY: 0
diff --git a/Assets/Animations/OrcRider/Hurt.anim b/Assets/Animations/OrcRider/Hurt.anim
index 4e4462cc..e8d04790 100644
--- a/Assets/Animations/OrcRider/Hurt.anim
+++ b/Assets/Animations/OrcRider/Hurt.anim
@@ -21,15 +21,13 @@ AnimationClip:
   - serializedVersion: 2
     curve:
     - time: 0
-      value: {fileID: -6259615744941935208, guid: 78f2309cbc878bd4c93862b47ae40ebf, type: 3}
-    - time: 0.099999905
-      value: {fileID: -6320779589914060938, guid: 78f2309cbc878bd4c93862b47ae40ebf, type: 3}
-    - time: 0.20000029
-      value: {fileID: 1221691598633469670, guid: 78f2309cbc878bd4c93862b47ae40ebf, type: 3}
-    - time: 0.3000002
-      value: {fileID: -1368790988085550589, guid: 78f2309cbc878bd4c93862b47ae40ebf, type: 3}
-    - time: 0.3900002
-      value: {fileID: -1368790988085550589, guid: 78f2309cbc878bd4c93862b47ae40ebf, type: 3}
+      value: {fileID: -7968779640083838834, guid: 68469bddce4ee9146ae8e66a85808c17, type: 3}
+    - time: 0.13
+      value: {fileID: -486091560615662378, guid: 68469bddce4ee9146ae8e66a85808c17, type: 3}
+    - time: 0.27
+      value: {fileID: -895491228106729965, guid: 68469bddce4ee9146ae8e66a85808c17, type: 3}
+    - time: 0.4
+      value: {fileID: -1731499667720825828, guid: 68469bddce4ee9146ae8e66a85808c17, type: 3}
     attribute: m_Sprite
     path: 
     classID: 212
@@ -41,19 +39,32 @@ AnimationClip:
     m_Center: {x: 0, y: 0, z: 0}
     m_Extent: {x: 0, y: 0, z: 0}
   m_ClipBindingConstant:
-    genericBindings: []
-    pptrCurveMapping: []
+    genericBindings:
+    - serializedVersion: 2
+      path: 0
+      attribute: 0
+      script: {fileID: 0}
+      typeID: 212
+      customType: 23
+      isPPtrCurve: 1
+      isIntCurve: 0
+      isSerializeReferenceCurve: 0
+    pptrCurveMapping:
+    - {fileID: -7968779640083838834, guid: 68469bddce4ee9146ae8e66a85808c17, type: 3}
+    - {fileID: -486091560615662378, guid: 68469bddce4ee9146ae8e66a85808c17, type: 3}
+    - {fileID: -895491228106729965, guid: 68469bddce4ee9146ae8e66a85808c17, type: 3}
+    - {fileID: -1731499667720825828, guid: 68469bddce4ee9146ae8e66a85808c17, type: 3}
   m_AnimationClipSettings:
     serializedVersion: 2
     m_AdditiveReferencePoseClip: {fileID: 0}
     m_AdditiveReferencePoseTime: 0
     m_StartTime: 0
-    m_StopTime: 0.40000018
+    m_StopTime: 0.41
     m_OrientationOffsetY: 0
     m_Level: 0
     m_CycleOffset: 0
     m_HasAdditiveReferencePose: 0
-    m_LoopTime: 0
+    m_LoopTime: 1
     m_LoopBlend: 0
     m_LoopBlendOrientation: 0
     m_LoopBlendPositionY: 0
diff --git a/Assets/Animations/OrcRider/Idle.anim b/Assets/Animations/OrcRider/Idle.anim
index dcd08210..02bc1349 100644
--- a/Assets/Animations/OrcRider/Idle.anim
+++ b/Assets/Animations/OrcRider/Idle.anim
@@ -23,14 +23,16 @@ AnimationClip:
     - time: 0
       value: {fileID: -6967218827304948987, guid: bb6dbe7158c9e434cb9e8624f1f81d6c, type: 3}
     - time: 0.083333336
-      value: {fileID: -2582689681796817123, guid: bb6dbe7158c9e434cb9e8624f1f81d6c, type: 3}
+      value: {fileID: -6967218827304948987, guid: bb6dbe7158c9e434cb9e8624f1f81d6c, type: 3}
     - time: 0.16666667
-      value: {fileID: 5239070887845672902, guid: bb6dbe7158c9e434cb9e8624f1f81d6c, type: 3}
+      value: {fileID: -2582689681796817123, guid: bb6dbe7158c9e434cb9e8624f1f81d6c, type: 3}
     - time: 0.25
-      value: {fileID: 8711236559198195290, guid: bb6dbe7158c9e434cb9e8624f1f81d6c, type: 3}
+      value: {fileID: 5239070887845672902, guid: bb6dbe7158c9e434cb9e8624f1f81d6c, type: 3}
     - time: 0.33333334
-      value: {fileID: -5612623993386223237, guid: bb6dbe7158c9e434cb9e8624f1f81d6c, type: 3}
+      value: {fileID: 8711236559198195290, guid: bb6dbe7158c9e434cb9e8624f1f81d6c, type: 3}
     - time: 0.41666666
+      value: {fileID: -5612623993386223237, guid: bb6dbe7158c9e434cb9e8624f1f81d6c, type: 3}
+    - time: 0.5
       value: {fileID: -600239269196264424, guid: bb6dbe7158c9e434cb9e8624f1f81d6c, type: 3}
     attribute: m_Sprite
     path: 
@@ -55,6 +57,7 @@ AnimationClip:
       isSerializeReferenceCurve: 0
     pptrCurveMapping:
     - {fileID: -6967218827304948987, guid: bb6dbe7158c9e434cb9e8624f1f81d6c, type: 3}
+    - {fileID: -6967218827304948987, guid: bb6dbe7158c9e434cb9e8624f1f81d6c, type: 3}
     - {fileID: -2582689681796817123, guid: bb6dbe7158c9e434cb9e8624f1f81d6c, type: 3}
     - {fileID: 5239070887845672902, guid: bb6dbe7158c9e434cb9e8624f1f81d6c, type: 3}
     - {fileID: 8711236559198195290, guid: bb6dbe7158c9e434cb9e8624f1f81d6c, type: 3}
@@ -65,7 +68,7 @@ AnimationClip:
     m_AdditiveReferencePoseClip: {fileID: 0}
     m_AdditiveReferencePoseTime: 0
     m_StartTime: 0
-    m_StopTime: 0.5
+    m_StopTime: 0.5833333
     m_OrientationOffsetY: 0
     m_Level: 0
     m_CycleOffset: 0
diff --git a/Assets/Animations/OrcRider/Orc rider-Idle_0.controller b/Assets/Animations/OrcRider/Orc rider-Idle_0.controller
index 226f6f84..55d0a3a9 100644
--- a/Assets/Animations/OrcRider/Orc rider-Idle_0.controller	
+++ b/Assets/Animations/OrcRider/Orc rider-Idle_0.controller	
@@ -36,7 +36,10 @@ AnimatorStateTransition:
   m_PrefabInstance: {fileID: 0}
   m_PrefabAsset: {fileID: 0}
   m_Name: 
-  m_Conditions: []
+  m_Conditions:
+  - m_ConditionMode: 1
+    m_ConditionEvent: IsWalking
+    m_EventTreshold: 0
   m_DstStateMachine: {fileID: 0}
   m_DstState: {fileID: 2068634877856680688}
   m_Solo: 0
@@ -46,7 +49,7 @@ AnimatorStateTransition:
   m_TransitionDuration: 0.25
   m_TransitionOffset: 0
   m_ExitTime: 0.5
-  m_HasExitTime: 1
+  m_HasExitTime: 0
   m_HasFixedDuration: 1
   m_InterruptionSource: 0
   m_OrderedInterruption: 1
@@ -68,7 +71,7 @@ AnimatorStateMachine:
     m_Position: {x: 380, y: 120, z: 0}
   - serializedVersion: 1
     m_State: {fileID: 4924873775655093501}
-    m_Position: {x: 520, y: 350, z: 0}
+    m_Position: {x: 470, y: 370, z: 0}
   - serializedVersion: 1
     m_State: {fileID: 6039973887249208931}
     m_Position: {x: 220, y: 360, z: 0}
@@ -92,7 +95,10 @@ AnimatorStateTransition:
   m_PrefabInstance: {fileID: 0}
   m_PrefabAsset: {fileID: 0}
   m_Name: 
-  m_Conditions: []
+  m_Conditions:
+  - m_ConditionMode: 1
+    m_ConditionEvent: IsWalking
+    m_EventTreshold: 0
   m_DstStateMachine: {fileID: 0}
   m_DstState: {fileID: 2068634877856680688}
   m_Solo: 0
@@ -101,7 +107,7 @@ AnimatorStateTransition:
   serializedVersion: 3
   m_TransitionDuration: 0.25
   m_TransitionOffset: 0
-  m_ExitTime: 0.3750003
+  m_ExitTime: 0.00625006
   m_HasExitTime: 1
   m_HasFixedDuration: 1
   m_InterruptionSource: 0
@@ -114,7 +120,10 @@ AnimatorStateTransition:
   m_PrefabInstance: {fileID: 0}
   m_PrefabAsset: {fileID: 0}
   m_Name: 
-  m_Conditions: []
+  m_Conditions:
+  - m_ConditionMode: 2
+    m_ConditionEvent: IsWalking
+    m_EventTreshold: 0
   m_DstStateMachine: {fileID: 0}
   m_DstState: {fileID: -2628642527416038780}
   m_Solo: 0
@@ -123,8 +132,33 @@ AnimatorStateTransition:
   serializedVersion: 3
   m_TransitionDuration: 0.25
   m_TransitionOffset: 0
-  m_ExitTime: 0.6875
-  m_HasExitTime: 1
+  m_ExitTime: 1
+  m_HasExitTime: 0
+  m_HasFixedDuration: 1
+  m_InterruptionSource: 0
+  m_OrderedInterruption: 1
+  m_CanTransitionToSelf: 1
+--- !u!1101 &-2940616943942556122
+AnimatorStateTransition:
+  m_ObjectHideFlags: 1
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_Name: 
+  m_Conditions:
+  - m_ConditionMode: 1
+    m_ConditionEvent: IsCharging
+    m_EventTreshold: 0
+  m_DstStateMachine: {fileID: 0}
+  m_DstState: {fileID: 4924873775655093501}
+  m_Solo: 0
+  m_Mute: 0
+  m_IsExit: 0
+  serializedVersion: 3
+  m_TransitionDuration: 0
+  m_TransitionOffset: 0
+  m_ExitTime: 0
+  m_HasExitTime: 0
   m_HasFixedDuration: 1
   m_InterruptionSource: 0
   m_OrderedInterruption: 1
@@ -136,17 +170,20 @@ AnimatorStateTransition:
   m_PrefabInstance: {fileID: 0}
   m_PrefabAsset: {fileID: 0}
   m_Name: 
-  m_Conditions: []
+  m_Conditions:
+  - m_ConditionMode: 1
+    m_ConditionEvent: IsAttacking
+    m_EventTreshold: 0
   m_DstStateMachine: {fileID: 0}
   m_DstState: {fileID: -7280255100376907123}
   m_Solo: 0
   m_Mute: 0
   m_IsExit: 0
   serializedVersion: 3
-  m_TransitionDuration: 0.25
+  m_TransitionDuration: 0
   m_TransitionOffset: 0
-  m_ExitTime: 0.5
-  m_HasExitTime: 1
+  m_ExitTime: 0
+  m_HasExitTime: 0
   m_HasFixedDuration: 1
   m_InterruptionSource: 0
   m_OrderedInterruption: 1
@@ -165,6 +202,7 @@ AnimatorState:
   - {fileID: -7198212194954978919}
   - {fileID: 4181954313856359061}
   - {fileID: -2894305656748642439}
+  - {fileID: -2940616943942556122}
   m_StateMachineBehaviours: []
   m_Position: {x: 50, y: 50, z: 0}
   m_IKOnFeet: 0
@@ -187,7 +225,10 @@ AnimatorStateTransition:
   m_PrefabInstance: {fileID: 0}
   m_PrefabAsset: {fileID: 0}
   m_Name: 
-  m_Conditions: []
+  m_Conditions:
+  - m_ConditionMode: 2
+    m_ConditionEvent: IsWalking
+    m_EventTreshold: 0
   m_DstStateMachine: {fileID: 0}
   m_DstState: {fileID: -2628642527416038780}
   m_Solo: 0
@@ -196,7 +237,7 @@ AnimatorStateTransition:
   serializedVersion: 3
   m_TransitionDuration: 0.25
   m_TransitionOffset: 0
-  m_ExitTime: 0.3750003
+  m_ExitTime: 0.0625006
   m_HasExitTime: 1
   m_HasFixedDuration: 1
   m_InterruptionSource: 0
@@ -209,17 +250,23 @@ AnimatorStateTransition:
   m_PrefabInstance: {fileID: 0}
   m_PrefabAsset: {fileID: 0}
   m_Name: 
-  m_Conditions: []
+  m_Conditions:
+  - m_ConditionMode: 1
+    m_ConditionEvent: IsWalking
+    m_EventTreshold: 0
+  - m_ConditionMode: 2
+    m_ConditionEvent: IsAttacking
+    m_EventTreshold: 0
   m_DstStateMachine: {fileID: 0}
   m_DstState: {fileID: 2068634877856680688}
   m_Solo: 0
   m_Mute: 0
   m_IsExit: 0
   serializedVersion: 3
-  m_TransitionDuration: 0.25
+  m_TransitionDuration: 0
   m_TransitionOffset: 0
-  m_ExitTime: 0.7222222
-  m_HasExitTime: 1
+  m_ExitTime: 0
+  m_HasExitTime: 0
   m_HasFixedDuration: 1
   m_InterruptionSource: 0
   m_OrderedInterruption: 1
@@ -245,12 +292,18 @@ AnimatorController:
     m_DefaultInt: 0
     m_DefaultBool: 0
     m_Controller: {fileID: 9100000}
-  - m_Name: Charge
+  - m_Name: Hit
     m_Type: 9
     m_DefaultFloat: 0
     m_DefaultInt: 0
     m_DefaultBool: 0
     m_Controller: {fileID: 9100000}
+  - m_Name: IsCharging
+    m_Type: 4
+    m_DefaultFloat: 0
+    m_DefaultInt: 0
+    m_DefaultBool: 0
+    m_Controller: {fileID: 9100000}
   m_AnimatorLayers:
   - serializedVersion: 5
     m_Name: Base Layer
@@ -271,17 +324,48 @@ AnimatorStateTransition:
   m_PrefabInstance: {fileID: 0}
   m_PrefabAsset: {fileID: 0}
   m_Name: 
-  m_Conditions: []
+  m_Conditions:
+  - m_ConditionMode: 1
+    m_ConditionEvent: Hit
+    m_EventTreshold: 0
   m_DstStateMachine: {fileID: 0}
   m_DstState: {fileID: 6039973887249208931}
   m_Solo: 0
   m_Mute: 0
   m_IsExit: 0
   serializedVersion: 3
-  m_TransitionDuration: 0.25
+  m_TransitionDuration: 0
   m_TransitionOffset: 0
-  m_ExitTime: 0.7222222
-  m_HasExitTime: 1
+  m_ExitTime: 0
+  m_HasExitTime: 0
+  m_HasFixedDuration: 1
+  m_InterruptionSource: 0
+  m_OrderedInterruption: 1
+  m_CanTransitionToSelf: 1
+--- !u!1101 &1231874722676120472
+AnimatorStateTransition:
+  m_ObjectHideFlags: 1
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_Name: 
+  m_Conditions:
+  - m_ConditionMode: 1
+    m_ConditionEvent: IsWalking
+    m_EventTreshold: 0
+  - m_ConditionMode: 2
+    m_ConditionEvent: IsCharging
+    m_EventTreshold: 0
+  m_DstStateMachine: {fileID: 0}
+  m_DstState: {fileID: 2068634877856680688}
+  m_Solo: 0
+  m_Mute: 0
+  m_IsExit: 0
+  serializedVersion: 3
+  m_TransitionDuration: 0
+  m_TransitionOffset: 0
+  m_ExitTime: 0
+  m_HasExitTime: 0
   m_HasFixedDuration: 1
   m_InterruptionSource: 0
   m_OrderedInterruption: 1
@@ -300,6 +384,7 @@ AnimatorState:
   - {fileID: -5225281819362542950}
   - {fileID: 7369310728342684205}
   - {fileID: 4249349125466386331}
+  - {fileID: 4046946031557000416}
   m_StateMachineBehaviours: []
   m_Position: {x: 50, y: 50, z: 0}
   m_IKOnFeet: 0
@@ -315,6 +400,31 @@ AnimatorState:
   m_MirrorParameter: 
   m_CycleOffsetParameter: 
   m_TimeParameter: 
+--- !u!1101 &4046946031557000416
+AnimatorStateTransition:
+  m_ObjectHideFlags: 1
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_Name: 
+  m_Conditions:
+  - m_ConditionMode: 1
+    m_ConditionEvent: IsCharging
+    m_EventTreshold: 0
+  m_DstStateMachine: {fileID: 0}
+  m_DstState: {fileID: 4924873775655093501}
+  m_Solo: 0
+  m_Mute: 0
+  m_IsExit: 0
+  serializedVersion: 3
+  m_TransitionDuration: 0
+  m_TransitionOffset: 0
+  m_ExitTime: 0
+  m_HasExitTime: 0
+  m_HasFixedDuration: 1
+  m_InterruptionSource: 0
+  m_OrderedInterruption: 1
+  m_CanTransitionToSelf: 1
 --- !u!1101 &4181954313856359061
 AnimatorStateTransition:
   m_ObjectHideFlags: 1
@@ -322,7 +432,10 @@ AnimatorStateTransition:
   m_PrefabInstance: {fileID: 0}
   m_PrefabAsset: {fileID: 0}
   m_Name: 
-  m_Conditions: []
+  m_Conditions:
+  - m_ConditionMode: 1
+    m_ConditionEvent: Hit
+    m_EventTreshold: 0
   m_DstStateMachine: {fileID: 0}
   m_DstState: {fileID: 6039973887249208931}
   m_Solo: 0
@@ -331,8 +444,8 @@ AnimatorStateTransition:
   serializedVersion: 3
   m_TransitionDuration: 0.25
   m_TransitionOffset: 0
-  m_ExitTime: 0.5
-  m_HasExitTime: 1
+  m_ExitTime: 0
+  m_HasExitTime: 0
   m_HasFixedDuration: 1
   m_InterruptionSource: 0
   m_OrderedInterruption: 1
@@ -344,17 +457,20 @@ AnimatorStateTransition:
   m_PrefabInstance: {fileID: 0}
   m_PrefabAsset: {fileID: 0}
   m_Name: 
-  m_Conditions: []
+  m_Conditions:
+  - m_ConditionMode: 1
+    m_ConditionEvent: IsWalking
+    m_EventTreshold: 0
   m_DstStateMachine: {fileID: 0}
   m_DstState: {fileID: -7280255100376907123}
   m_Solo: 0
   m_Mute: 0
   m_IsExit: 0
   serializedVersion: 3
-  m_TransitionDuration: 0.25
+  m_TransitionDuration: 0
   m_TransitionOffset: 0
-  m_ExitTime: 0.6875
-  m_HasExitTime: 1
+  m_ExitTime: 0
+  m_HasExitTime: 0
   m_HasFixedDuration: 1
   m_InterruptionSource: 0
   m_OrderedInterruption: 1
@@ -369,7 +485,9 @@ AnimatorState:
   m_Name: Charge
   m_Speed: 1
   m_CycleOffset: 0
-  m_Transitions: []
+  m_Transitions:
+  - {fileID: 6708706408237427116}
+  - {fileID: 1231874722676120472}
   m_StateMachineBehaviours: []
   m_Position: {x: 50, y: 50, z: 0}
   m_IKOnFeet: 0
@@ -420,17 +538,51 @@ AnimatorStateTransition:
   m_PrefabInstance: {fileID: 0}
   m_PrefabAsset: {fileID: 0}
   m_Name: 
-  m_Conditions: []
+  m_Conditions:
+  - m_ConditionMode: 2
+    m_ConditionEvent: IsAttacking
+    m_EventTreshold: 0
+  - m_ConditionMode: 2
+    m_ConditionEvent: IsWalking
+    m_EventTreshold: 0
   m_DstStateMachine: {fileID: 0}
   m_DstState: {fileID: -2628642527416038780}
   m_Solo: 0
   m_Mute: 0
   m_IsExit: 0
   serializedVersion: 3
-  m_TransitionDuration: 0.25
+  m_TransitionDuration: 0
   m_TransitionOffset: 0
-  m_ExitTime: 0.7222222
-  m_HasExitTime: 1
+  m_ExitTime: 0
+  m_HasExitTime: 0
+  m_HasFixedDuration: 1
+  m_InterruptionSource: 0
+  m_OrderedInterruption: 1
+  m_CanTransitionToSelf: 1
+--- !u!1101 &6708706408237427116
+AnimatorStateTransition:
+  m_ObjectHideFlags: 1
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_Name: 
+  m_Conditions:
+  - m_ConditionMode: 2
+    m_ConditionEvent: IsCharging
+    m_EventTreshold: 0
+  - m_ConditionMode: 2
+    m_ConditionEvent: IsWalking
+    m_EventTreshold: 0
+  m_DstStateMachine: {fileID: 0}
+  m_DstState: {fileID: -2628642527416038780}
+  m_Solo: 0
+  m_Mute: 0
+  m_IsExit: 0
+  serializedVersion: 3
+  m_TransitionDuration: 0
+  m_TransitionOffset: 0
+  m_ExitTime: 0
+  m_HasExitTime: 0
   m_HasFixedDuration: 1
   m_InterruptionSource: 0
   m_OrderedInterruption: 1
@@ -451,8 +603,8 @@ AnimatorStateTransition:
   serializedVersion: 3
   m_TransitionDuration: 0.25
   m_TransitionOffset: 0
-  m_ExitTime: 0.6875
-  m_HasExitTime: 1
+  m_ExitTime: 0
+  m_HasExitTime: 0
   m_HasFixedDuration: 1
   m_InterruptionSource: 0
   m_OrderedInterruption: 1
diff --git a/Assets/Animations/OrcRider/Walk.anim b/Assets/Animations/OrcRider/Walk.anim
index 6b9ca3c2..f6e8b65b 100644
--- a/Assets/Animations/OrcRider/Walk.anim
+++ b/Assets/Animations/OrcRider/Walk.anim
@@ -21,23 +21,21 @@ AnimationClip:
   - serializedVersion: 2
     curve:
     - time: 0
-      value: {fileID: 1613032658754914055, guid: 78f2309cbc878bd4c93862b47ae40ebf, type: 3}
-    - time: 0.099999964
-      value: {fileID: -8008706243214644141, guid: 78f2309cbc878bd4c93862b47ae40ebf, type: 3}
-    - time: 0.19999999
-      value: {fileID: 9091856105791666712, guid: 78f2309cbc878bd4c93862b47ae40ebf, type: 3}
-    - time: 0.29999995
-      value: {fileID: 7711893465970493904, guid: 78f2309cbc878bd4c93862b47ae40ebf, type: 3}
-    - time: 0.39999998
-      value: {fileID: -1154213702454624187, guid: 78f2309cbc878bd4c93862b47ae40ebf, type: 3}
-    - time: 0.5
-      value: {fileID: -4187954839699068451, guid: 78f2309cbc878bd4c93862b47ae40ebf, type: 3}
+      value: {fileID: 4762972965759509545, guid: 22b8e25471fa4db4d9433834f818b40a, type: 3}
+    - time: 0.1
+      value: {fileID: -467496773326053826, guid: 22b8e25471fa4db4d9433834f818b40a, type: 3}
+    - time: 0.21
+      value: {fileID: -4277353276443176154, guid: 22b8e25471fa4db4d9433834f818b40a, type: 3}
+    - time: 0.3
+      value: {fileID: 3971616282867014591, guid: 22b8e25471fa4db4d9433834f818b40a, type: 3}
+    - time: 0.4
+      value: {fileID: 1127462252342030170, guid: 22b8e25471fa4db4d9433834f818b40a, type: 3}
+    - time: 0.49
+      value: {fileID: 5749074350868987293, guid: 22b8e25471fa4db4d9433834f818b40a, type: 3}
     - time: 0.6
-      value: {fileID: 29602314246264429, guid: 78f2309cbc878bd4c93862b47ae40ebf, type: 3}
-    - time: 0.6999999
-      value: {fileID: -5163209844761367654, guid: 78f2309cbc878bd4c93862b47ae40ebf, type: 3}
-    - time: 0.78999996
-      value: {fileID: -5163209844761367654, guid: 78f2309cbc878bd4c93862b47ae40ebf, type: 3}
+      value: {fileID: -63334707547655146, guid: 22b8e25471fa4db4d9433834f818b40a, type: 3}
+    - time: 0.7
+      value: {fileID: 3572186884227599840, guid: 22b8e25471fa4db4d9433834f818b40a, type: 3}
     attribute: m_Sprite
     path: 
     classID: 212
@@ -49,19 +47,36 @@ AnimationClip:
     m_Center: {x: 0, y: 0, z: 0}
     m_Extent: {x: 0, y: 0, z: 0}
   m_ClipBindingConstant:
-    genericBindings: []
-    pptrCurveMapping: []
+    genericBindings:
+    - serializedVersion: 2
+      path: 0
+      attribute: 0
+      script: {fileID: 0}
+      typeID: 212
+      customType: 23
+      isPPtrCurve: 1
+      isIntCurve: 0
+      isSerializeReferenceCurve: 0
+    pptrCurveMapping:
+    - {fileID: 4762972965759509545, guid: 22b8e25471fa4db4d9433834f818b40a, type: 3}
+    - {fileID: -467496773326053826, guid: 22b8e25471fa4db4d9433834f818b40a, type: 3}
+    - {fileID: -4277353276443176154, guid: 22b8e25471fa4db4d9433834f818b40a, type: 3}
+    - {fileID: 3971616282867014591, guid: 22b8e25471fa4db4d9433834f818b40a, type: 3}
+    - {fileID: 1127462252342030170, guid: 22b8e25471fa4db4d9433834f818b40a, type: 3}
+    - {fileID: 5749074350868987293, guid: 22b8e25471fa4db4d9433834f818b40a, type: 3}
+    - {fileID: -63334707547655146, guid: 22b8e25471fa4db4d9433834f818b40a, type: 3}
+    - {fileID: 3572186884227599840, guid: 22b8e25471fa4db4d9433834f818b40a, type: 3}
   m_AnimationClipSettings:
     serializedVersion: 2
     m_AdditiveReferencePoseClip: {fileID: 0}
     m_AdditiveReferencePoseTime: 0
     m_StartTime: 0
-    m_StopTime: 0.79999995
+    m_StopTime: 0.71
     m_OrientationOffsetY: 0
     m_Level: 0
     m_CycleOffset: 0
     m_HasAdditiveReferencePose: 0
-    m_LoopTime: 0
+    m_LoopTime: 1
     m_LoopBlend: 0
     m_LoopBlendOrientation: 0
     m_LoopBlendPositionY: 0
diff --git a/Assets/Scenes/Map1.unity b/Assets/Scenes/Map1.unity
index fd13c824..5f16ab4c 100644
--- a/Assets/Scenes/Map1.unity
+++ b/Assets/Scenes/Map1.unity
@@ -58132,6 +58132,63 @@ PrefabInstance:
     m_AddedGameObjects: []
     m_AddedComponents: []
   m_SourcePrefab: {fileID: 100100000, guid: 84d669757910f1f488023c92e71fcad5, type: 3}
+--- !u!1001 &7016840811299150641
+PrefabInstance:
+  m_ObjectHideFlags: 0
+  serializedVersion: 2
+  m_Modification:
+    serializedVersion: 3
+    m_TransformParent: {fileID: 0}
+    m_Modifications:
+    - target: {fileID: 4491971134594873313, guid: ab084fbaadf7cdc45abd22413fdf49d3, type: 3}
+      propertyPath: m_Name
+      value: OrcRider
+      objectReference: {fileID: 0}
+    - target: {fileID: 7912430587132915113, guid: ab084fbaadf7cdc45abd22413fdf49d3, type: 3}
+      propertyPath: m_LocalPosition.x
+      value: 0.95
+      objectReference: {fileID: 0}
+    - target: {fileID: 7912430587132915113, guid: ab084fbaadf7cdc45abd22413fdf49d3, type: 3}
+      propertyPath: m_LocalPosition.y
+      value: 2.68
+      objectReference: {fileID: 0}
+    - target: {fileID: 7912430587132915113, guid: ab084fbaadf7cdc45abd22413fdf49d3, type: 3}
+      propertyPath: m_LocalPosition.z
+      value: 0.06559052
+      objectReference: {fileID: 0}
+    - target: {fileID: 7912430587132915113, guid: ab084fbaadf7cdc45abd22413fdf49d3, type: 3}
+      propertyPath: m_LocalRotation.w
+      value: 1
+      objectReference: {fileID: 0}
+    - target: {fileID: 7912430587132915113, guid: ab084fbaadf7cdc45abd22413fdf49d3, type: 3}
+      propertyPath: m_LocalRotation.x
+      value: 0
+      objectReference: {fileID: 0}
+    - target: {fileID: 7912430587132915113, guid: ab084fbaadf7cdc45abd22413fdf49d3, type: 3}
+      propertyPath: m_LocalRotation.y
+      value: 0
+      objectReference: {fileID: 0}
+    - target: {fileID: 7912430587132915113, guid: ab084fbaadf7cdc45abd22413fdf49d3, type: 3}
+      propertyPath: m_LocalRotation.z
+      value: 0
+      objectReference: {fileID: 0}
+    - target: {fileID: 7912430587132915113, guid: ab084fbaadf7cdc45abd22413fdf49d3, type: 3}
+      propertyPath: m_LocalEulerAnglesHint.x
+      value: 0
+      objectReference: {fileID: 0}
+    - target: {fileID: 7912430587132915113, guid: ab084fbaadf7cdc45abd22413fdf49d3, type: 3}
+      propertyPath: m_LocalEulerAnglesHint.y
+      value: 0
+      objectReference: {fileID: 0}
+    - target: {fileID: 7912430587132915113, guid: ab084fbaadf7cdc45abd22413fdf49d3, type: 3}
+      propertyPath: m_LocalEulerAnglesHint.z
+      value: 0
+      objectReference: {fileID: 0}
+    m_RemovedComponents: []
+    m_RemovedGameObjects: []
+    m_AddedGameObjects: []
+    m_AddedComponents: []
+  m_SourcePrefab: {fileID: 100100000, guid: ab084fbaadf7cdc45abd22413fdf49d3, type: 3}
 --- !u!1001 &7109474815243934248
 PrefabInstance:
   m_ObjectHideFlags: 0
@@ -58624,3 +58681,4 @@ SceneRoots:
   - {fileID: 225949324}
   - {fileID: 269859199}
   - {fileID: 1768831311}
+  - {fileID: 7016840811299150641}
diff --git a/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider with shadows/Orc rider-Attack02.png.meta b/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider with shadows/Orc rider-Attack02.png.meta
index 4d4ccc2e..6aced966 100644
--- a/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider with shadows/Orc rider-Attack02.png.meta	
+++ b/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider with shadows/Orc rider-Attack02.png.meta	
@@ -139,10 +139,10 @@ TextureImporter:
       name: Orc rider-Attack02_0
       rect:
         serializedVersion: 2
-        x: 36
-        y: 39
-        width: 34
-        height: 33
+        x: 0
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -161,10 +161,10 @@ TextureImporter:
       name: Orc rider-Attack02_1
       rect:
         serializedVersion: 2
-        x: 136
-        y: 39
-        width: 33
-        height: 34
+        x: 100
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -183,10 +183,10 @@ TextureImporter:
       name: Orc rider-Attack02_2
       rect:
         serializedVersion: 2
-        x: 236
-        y: 39
-        width: 33
-        height: 38
+        x: 200
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -205,10 +205,10 @@ TextureImporter:
       name: Orc rider-Attack02_3
       rect:
         serializedVersion: 2
-        x: 336
-        y: 39
-        width: 34
-        height: 45
+        x: 300
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -227,10 +227,10 @@ TextureImporter:
       name: Orc rider-Attack02_4
       rect:
         serializedVersion: 2
-        x: 436
-        y: 39
-        width: 33
-        height: 33
+        x: 400
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -249,10 +249,10 @@ TextureImporter:
       name: Orc rider-Attack02_5
       rect:
         serializedVersion: 2
-        x: 455
-        y: 67
-        width: 32
-        height: 24
+        x: 500
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -271,10 +271,10 @@ TextureImporter:
       name: Orc rider-Attack02_6
       rect:
         serializedVersion: 2
-        x: 458
-        y: 63
-        width: 10
-        height: 10
+        x: 600
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -293,10 +293,10 @@ TextureImporter:
       name: Orc rider-Attack02_7
       rect:
         serializedVersion: 2
-        x: 536
-        y: 39
-        width: 38
-        height: 32
+        x: 700
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -315,10 +315,10 @@ TextureImporter:
       name: Orc rider-Attack02_8
       rect:
         serializedVersion: 2
-        x: 573
-        y: 26
-        width: 24
-        height: 43
+        x: 800
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -333,99 +333,11 @@ TextureImporter:
       indices: 
       edges: []
       weights: []
-    - serializedVersion: 2
-      name: Orc rider-Attack02_9
-      rect:
-        serializedVersion: 2
-        x: 636
-        y: 39
-        width: 33
-        height: 32
-      alignment: 0
-      pivot: {x: 0, y: 0}
-      border: {x: 0, y: 0, z: 0, w: 0}
-      customData: 
-      outline: []
-      physicsShape: []
-      tessellationDetail: -1
-      bones: []
-      spriteID: 1e9610780879317d0800000000000000
-      internalID: -2948846752720918047
-      vertices: []
-      indices: 
-      edges: []
-      weights: []
-    - serializedVersion: 2
-      name: Orc rider-Attack02_10
-      rect:
-        serializedVersion: 2
-        x: 736
-        y: 39
-        width: 51
-        height: 32
-      alignment: 0
-      pivot: {x: 0, y: 0}
-      border: {x: 0, y: 0, z: 0, w: 0}
-      customData: 
-      outline: []
-      physicsShape: []
-      tessellationDetail: -1
-      bones: []
-      spriteID: e1f9c8001aa986980800000000000000
-      internalID: -8545410276645888226
-      vertices: []
-      indices: 
-      edges: []
-      weights: []
-    - serializedVersion: 2
-      name: Orc rider-Attack02_11
-      rect:
-        serializedVersion: 2
-        x: 836
-        y: 39
-        width: 35
-        height: 33
-      alignment: 0
-      pivot: {x: 0, y: 0}
-      border: {x: 0, y: 0, z: 0, w: 0}
-      customData: 
-      outline: []
-      physicsShape: []
-      tessellationDetail: -1
-      bones: []
-      spriteID: f353062f167fdce50800000000000000
-      internalID: 6831388209841190207
-      vertices: []
-      indices: 
-      edges: []
-      weights: []
-    - serializedVersion: 2
-      name: Orc rider-Attack02_12
-      rect:
-        serializedVersion: 2
-        x: 673
-        y: 29
-        width: 20
-        height: 34
-      alignment: 0
-      pivot: {x: 0, y: 0}
-      border: {x: 0, y: 0, z: 0, w: 0}
-      customData: 
-      outline: []
-      physicsShape: []
-      tessellationDetail: -1
-      bones: []
-      spriteID: 63b85931bc37bd710800000000000000
-      internalID: 1719094998816099126
-      vertices: []
-      indices: 
-      edges: []
-      weights: []
     outline: []
     customData: 
     physicsShape: []
     bones: []
-    spriteID: 
+    spriteID: 48453798a020d3e4d85bac0605824a6a
     internalID: 0
     vertices: []
     indices: 
@@ -437,9 +349,6 @@ TextureImporter:
     nameFileIdTable:
       Orc rider-Attack02_0: -4805291195941671494
       Orc rider-Attack02_1: -898491107783281163
-      Orc rider-Attack02_10: -8545410276645888226
-      Orc rider-Attack02_11: 6831388209841190207
-      Orc rider-Attack02_12: 1719094998816099126
       Orc rider-Attack02_2: 2851206345125205499
       Orc rider-Attack02_3: 5773026405177017699
       Orc rider-Attack02_4: -3239606791055260546
@@ -447,7 +356,6 @@ TextureImporter:
       Orc rider-Attack02_6: 837233935513997722
       Orc rider-Attack02_7: 7414343374642175032
       Orc rider-Attack02_8: 6936758103650135619
-      Orc rider-Attack02_9: -2948846752720918047
   mipmapLimitGroupName: 
   pSDRemoveMatte: 0
   userData: 
diff --git a/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider with shadows/Orc rider-Attack03.png.meta b/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider with shadows/Orc rider-Attack03.png.meta
index 72869c90..735fa5ef 100644
--- a/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider with shadows/Orc rider-Attack03.png.meta	
+++ b/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider with shadows/Orc rider-Attack03.png.meta	
@@ -133,10 +133,10 @@ TextureImporter:
       name: Orc rider-Attack03_0
       rect:
         serializedVersion: 2
-        x: 36
-        y: 39
-        width: 34
-        height: 33
+        x: 0
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -155,10 +155,10 @@ TextureImporter:
       name: Orc rider-Attack03_1
       rect:
         serializedVersion: 2
-        x: 136
-        y: 39
-        width: 33
-        height: 34
+        x: 100
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -177,10 +177,10 @@ TextureImporter:
       name: Orc rider-Attack03_2
       rect:
         serializedVersion: 2
-        x: 236
-        y: 39
-        width: 33
-        height: 38
+        x: 200
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -199,10 +199,10 @@ TextureImporter:
       name: Orc rider-Attack03_3
       rect:
         serializedVersion: 2
-        x: 336
-        y: 39
-        width: 34
-        height: 45
+        x: 300
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -221,10 +221,10 @@ TextureImporter:
       name: Orc rider-Attack03_4
       rect:
         serializedVersion: 2
-        x: 436
-        y: 39
-        width: 40
-        height: 43
+        x: 400
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -243,10 +243,10 @@ TextureImporter:
       name: Orc rider-Attack03_5
       rect:
         serializedVersion: 2
-        x: 536
-        y: 39
-        width: 59
-        height: 32
+        x: 500
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -265,10 +265,10 @@ TextureImporter:
       name: Orc rider-Attack03_6
       rect:
         serializedVersion: 2
-        x: 608
-        y: 39
-        width: 87
-        height: 32
+        x: 600
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -287,10 +287,10 @@ TextureImporter:
       name: Orc rider-Attack03_7
       rect:
         serializedVersion: 2
-        x: 708
-        y: 39
-        width: 61
-        height: 32
+        x: 700
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -309,10 +309,10 @@ TextureImporter:
       name: Orc rider-Attack03_8
       rect:
         serializedVersion: 2
-        x: 811
-        y: 39
-        width: 60
-        height: 32
+        x: 800
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -331,10 +331,10 @@ TextureImporter:
       name: Orc rider-Attack03_9
       rect:
         serializedVersion: 2
-        x: 936
-        y: 39
-        width: 59
-        height: 32
+        x: 900
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -353,10 +353,10 @@ TextureImporter:
       name: Orc rider-Attack03_10
       rect:
         serializedVersion: 2
-        x: 1036
-        y: 39
-        width: 33
-        height: 32
+        x: 1000
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -375,7 +375,7 @@ TextureImporter:
     customData: 
     physicsShape: []
     bones: []
-    spriteID: 
+    spriteID: 053df4cfad38b204caf962d7b5047db0
     internalID: 0
     vertices: []
     indices: 
diff --git a/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider with shadows/Orc rider-Block.png.meta b/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider with shadows/Orc rider-Block.png.meta
index 028d15ba..37884f83 100644
--- a/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider with shadows/Orc rider-Block.png.meta	
+++ b/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider with shadows/Orc rider-Block.png.meta	
@@ -112,10 +112,10 @@ TextureImporter:
       name: Orc rider-Block_0
       rect:
         serializedVersion: 2
-        x: 36
-        y: 39
-        width: 34
-        height: 33
+        x: 0
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -134,10 +134,10 @@ TextureImporter:
       name: Orc rider-Block_1
       rect:
         serializedVersion: 2
-        x: 136
-        y: 39
-        width: 33
-        height: 32
+        x: 100
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -156,10 +156,10 @@ TextureImporter:
       name: Orc rider-Block_2
       rect:
         serializedVersion: 2
-        x: 236
-        y: 39
-        width: 33
-        height: 32
+        x: 200
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -178,10 +178,10 @@ TextureImporter:
       name: Orc rider-Block_3
       rect:
         serializedVersion: 2
-        x: 336
-        y: 39
-        width: 34
-        height: 32
+        x: 300
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -200,7 +200,7 @@ TextureImporter:
     customData: 
     physicsShape: []
     bones: []
-    spriteID: 
+    spriteID: 02f1bbe93882a754fa9847b95c26b4cb
     internalID: 0
     vertices: []
     indices: 
diff --git a/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider with shadows/Orc rider-Death.png.meta b/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider with shadows/Orc rider-Death.png.meta
index 11eec98a..16dcdc5f 100644
--- a/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider with shadows/Orc rider-Death.png.meta	
+++ b/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider with shadows/Orc rider-Death.png.meta	
@@ -112,10 +112,10 @@ TextureImporter:
       name: Orc rider-Death_0
       rect:
         serializedVersion: 2
-        x: 36
-        y: 39
-        width: 34
-        height: 33
+        x: 0
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -134,10 +134,10 @@ TextureImporter:
       name: Orc rider-Death_1
       rect:
         serializedVersion: 2
-        x: 136
-        y: 39
-        width: 33
-        height: 32
+        x: 100
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -156,10 +156,10 @@ TextureImporter:
       name: Orc rider-Death_2
       rect:
         serializedVersion: 2
-        x: 232
-        y: 39
-        width: 45
-        height: 22
+        x: 200
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -178,10 +178,10 @@ TextureImporter:
       name: Orc rider-Death_3
       rect:
         serializedVersion: 2
-        x: 327
-        y: 39
-        width: 54
-        height: 17
+        x: 300
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -200,7 +200,7 @@ TextureImporter:
     customData: 
     physicsShape: []
     bones: []
-    spriteID: 
+    spriteID: cf73a74a6abc6bf4b869d0e5b987dfdf
     internalID: 0
     vertices: []
     indices: 
diff --git a/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider with shadows/Orc rider-Hurt.png.meta b/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider with shadows/Orc rider-Hurt.png.meta
index dec4b968..e09f0530 100644
--- a/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider with shadows/Orc rider-Hurt.png.meta	
+++ b/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider with shadows/Orc rider-Hurt.png.meta	
@@ -112,10 +112,10 @@ TextureImporter:
       name: Orc rider-Hurt_0
       rect:
         serializedVersion: 2
-        x: 36
-        y: 39
-        width: 34
-        height: 32
+        x: 0
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -134,10 +134,10 @@ TextureImporter:
       name: Orc rider-Hurt_1
       rect:
         serializedVersion: 2
-        x: 136
-        y: 39
-        width: 34
-        height: 32
+        x: 100
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -156,10 +156,10 @@ TextureImporter:
       name: Orc rider-Hurt_2
       rect:
         serializedVersion: 2
-        x: 236
-        y: 39
-        width: 33
-        height: 32
+        x: 200
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -178,10 +178,10 @@ TextureImporter:
       name: Orc rider-Hurt_3
       rect:
         serializedVersion: 2
-        x: 336
-        y: 39
-        width: 33
-        height: 32
+        x: 300
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -200,7 +200,7 @@ TextureImporter:
     customData: 
     physicsShape: []
     bones: []
-    spriteID: 
+    spriteID: 8ea3e34f06556b6438107c1a8f448b22
     internalID: 0
     vertices: []
     indices: 
diff --git a/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider with shadows/Orc rider-Walk.png.meta b/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider with shadows/Orc rider-Walk.png.meta
index f17f4368..4ff401dd 100644
--- a/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider with shadows/Orc rider-Walk.png.meta	
+++ b/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider with shadows/Orc rider-Walk.png.meta	
@@ -124,10 +124,10 @@ TextureImporter:
       name: Orc rider-Walk_0
       rect:
         serializedVersion: 2
-        x: 36
-        y: 39
-        width: 34
-        height: 33
+        x: 0
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -146,10 +146,10 @@ TextureImporter:
       name: Orc rider-Walk_1
       rect:
         serializedVersion: 2
-        x: 136
-        y: 39
-        width: 33
-        height: 34
+        x: 100
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -168,10 +168,10 @@ TextureImporter:
       name: Orc rider-Walk_2
       rect:
         serializedVersion: 2
-        x: 236
-        y: 39
-        width: 33
-        height: 33
+        x: 200
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -190,10 +190,10 @@ TextureImporter:
       name: Orc rider-Walk_3
       rect:
         serializedVersion: 2
-        x: 336
-        y: 39
-        width: 33
-        height: 32
+        x: 300
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -212,10 +212,10 @@ TextureImporter:
       name: Orc rider-Walk_4
       rect:
         serializedVersion: 2
-        x: 436
-        y: 39
-        width: 33
-        height: 33
+        x: 400
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -234,10 +234,10 @@ TextureImporter:
       name: Orc rider-Walk_5
       rect:
         serializedVersion: 2
-        x: 536
-        y: 39
-        width: 33
-        height: 33
+        x: 500
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -256,19 +256,19 @@ TextureImporter:
       name: Orc rider-Walk_6
       rect:
         serializedVersion: 2
-        x: 636
-        y: 39
-        width: 34
-        height: 32
+        x: 600
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
       customData: 
       outline: []
       physicsShape: []
-      tessellationDetail: -1
+      tessellationDetail: 0
       bones: []
-      spriteID: 6143ba64b6dfe1ff0800000000000000
+      spriteID: cc73fcde29c2fd2439b34712f06a61d4
       internalID: -63334707547655146
       vertices: []
       indices: 
@@ -278,19 +278,19 @@ TextureImporter:
       name: Orc rider-Walk_7
       rect:
         serializedVersion: 2
-        x: 736
-        y: 39
-        width: 35
-        height: 32
+        x: 700
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
       customData: 
       outline: []
       physicsShape: []
-      tessellationDetail: -1
+      tessellationDetail: 0
       bones: []
-      spriteID: 0ed02b18b05f29130800000000000000
+      spriteID: a1443d21748da9c4a995c64413ec7235
       internalID: 3572186884227599840
       vertices: []
       indices: 
@@ -300,7 +300,7 @@ TextureImporter:
     customData: 
     physicsShape: []
     bones: []
-    spriteID: 
+    spriteID: bc25c0f7128cd304ba1aeccbd2f03e76
     internalID: 0
     vertices: []
     indices: 
diff --git a/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider/Orc rider-Attack01.png.meta b/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider/Orc rider-Attack01.png.meta
index 8ef99698..957fb335 100644
--- a/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider/Orc rider-Attack01.png.meta	
+++ b/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider/Orc rider-Attack01.png.meta	
@@ -58,7 +58,7 @@ TextureImporter:
   maxTextureSize: 2048
   textureSettings:
     serializedVersion: 2
-    filterMode: 1
+    filterMode: 0
     aniso: 1
     mipBias: 0
     wrapU: 1
@@ -124,10 +124,10 @@ TextureImporter:
       name: Orc rider-Attack01_0
       rect:
         serializedVersion: 2
-        x: 36
-        y: 42
-        width: 34
-        height: 30
+        x: 0
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -146,10 +146,10 @@ TextureImporter:
       name: Orc rider-Attack01_1
       rect:
         serializedVersion: 2
-        x: 136
-        y: 42
-        width: 33
-        height: 29
+        x: 100
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -168,10 +168,10 @@ TextureImporter:
       name: Orc rider-Attack01_2
       rect:
         serializedVersion: 2
-        x: 236
-        y: 41
-        width: 32
-        height: 29
+        x: 200
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -190,10 +190,10 @@ TextureImporter:
       name: Orc rider-Attack01_3
       rect:
         serializedVersion: 2
-        x: 336
-        y: 42
-        width: 33
-        height: 30
+        x: 300
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -212,10 +212,10 @@ TextureImporter:
       name: Orc rider-Attack01_4
       rect:
         serializedVersion: 2
-        x: 436
-        y: 39
-        width: 45
-        height: 33
+        x: 400
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -234,10 +234,10 @@ TextureImporter:
       name: Orc rider-Attack01_5
       rect:
         serializedVersion: 2
-        x: 536
-        y: 42
-        width: 45
-        height: 30
+        x: 500
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -256,10 +256,10 @@ TextureImporter:
       name: Orc rider-Attack01_6
       rect:
         serializedVersion: 2
-        x: 636
-        y: 42
-        width: 44
-        height: 30
+        x: 600
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -278,10 +278,10 @@ TextureImporter:
       name: Orc rider-Attack01_7
       rect:
         serializedVersion: 2
-        x: 736
-        y: 42
-        width: 35
-        height: 30
+        x: 700
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -300,7 +300,7 @@ TextureImporter:
     customData: 
     physicsShape: []
     bones: []
-    spriteID: 
+    spriteID: 80b318d5bc43efe46ad11c66f576c171
     internalID: 0
     vertices: []
     indices: 
diff --git a/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider/Orc rider-Attack02.png.meta b/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider/Orc rider-Attack02.png.meta
index 3f49ad31..efbb0735 100644
--- a/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider/Orc rider-Attack02.png.meta	
+++ b/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider/Orc rider-Attack02.png.meta	
@@ -73,7 +73,7 @@ TextureImporter:
   maxTextureSize: 2048
   textureSettings:
     serializedVersion: 2
-    filterMode: 1
+    filterMode: 0
     aniso: 1
     mipBias: 0
     wrapU: 1
@@ -139,10 +139,10 @@ TextureImporter:
       name: Orc rider-Attack02_0
       rect:
         serializedVersion: 2
-        x: 36
-        y: 42
-        width: 34
-        height: 30
+        x: 0
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -161,10 +161,10 @@ TextureImporter:
       name: Orc rider-Attack02_1
       rect:
         serializedVersion: 2
-        x: 136
-        y: 42
-        width: 33
-        height: 31
+        x: 100
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -183,10 +183,10 @@ TextureImporter:
       name: Orc rider-Attack02_2
       rect:
         serializedVersion: 2
-        x: 236
-        y: 42
-        width: 33
-        height: 35
+        x: 200
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -205,10 +205,10 @@ TextureImporter:
       name: Orc rider-Attack02_3
       rect:
         serializedVersion: 2
-        x: 336
-        y: 42
-        width: 34
-        height: 42
+        x: 300
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -227,10 +227,10 @@ TextureImporter:
       name: Orc rider-Attack02_4
       rect:
         serializedVersion: 2
-        x: 436
-        y: 42
-        width: 33
-        height: 30
+        x: 400
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -249,10 +249,10 @@ TextureImporter:
       name: Orc rider-Attack02_5
       rect:
         serializedVersion: 2
-        x: 455
-        y: 67
-        width: 32
-        height: 24
+        x: 500
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -271,10 +271,10 @@ TextureImporter:
       name: Orc rider-Attack02_6
       rect:
         serializedVersion: 2
-        x: 458
-        y: 63
-        width: 10
-        height: 10
+        x: 600
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -293,10 +293,10 @@ TextureImporter:
       name: Orc rider-Attack02_7
       rect:
         serializedVersion: 2
-        x: 536
-        y: 42
-        width: 38
-        height: 29
+        x: 700
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -315,117 +315,29 @@ TextureImporter:
       name: Orc rider-Attack02_8
       rect:
         serializedVersion: 2
-        x: 573
-        y: 26
-        width: 24
-        height: 43
+        x: 800
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
       customData: 
       outline: []
       physicsShape: []
-      tessellationDetail: -1
+      tessellationDetail: 0
       bones: []
-      spriteID: 34ee0e42eb0544060800000000000000
+      spriteID: 28aba6ece19805243820346c99d1c151
       internalID: 6936758103650135619
       vertices: []
       indices: 
       edges: []
       weights: []
-    - serializedVersion: 2
-      name: Orc rider-Attack02_9
-      rect:
-        serializedVersion: 2
-        x: 636
-        y: 42
-        width: 33
-        height: 29
-      alignment: 0
-      pivot: {x: 0, y: 0}
-      border: {x: 0, y: 0, z: 0, w: 0}
-      customData: 
-      outline: []
-      physicsShape: []
-      tessellationDetail: -1
-      bones: []
-      spriteID: 1e9610780879317d0800000000000000
-      internalID: -2948846752720918047
-      vertices: []
-      indices: 
-      edges: []
-      weights: []
-    - serializedVersion: 2
-      name: Orc rider-Attack02_10
-      rect:
-        serializedVersion: 2
-        x: 736
-        y: 42
-        width: 51
-        height: 29
-      alignment: 0
-      pivot: {x: 0, y: 0}
-      border: {x: 0, y: 0, z: 0, w: 0}
-      customData: 
-      outline: []
-      physicsShape: []
-      tessellationDetail: -1
-      bones: []
-      spriteID: e1f9c8001aa986980800000000000000
-      internalID: -8545410276645888226
-      vertices: []
-      indices: 
-      edges: []
-      weights: []
-    - serializedVersion: 2
-      name: Orc rider-Attack02_11
-      rect:
-        serializedVersion: 2
-        x: 836
-        y: 42
-        width: 35
-        height: 30
-      alignment: 0
-      pivot: {x: 0, y: 0}
-      border: {x: 0, y: 0, z: 0, w: 0}
-      customData: 
-      outline: []
-      physicsShape: []
-      tessellationDetail: -1
-      bones: []
-      spriteID: f353062f167fdce50800000000000000
-      internalID: 6831388209841190207
-      vertices: []
-      indices: 
-      edges: []
-      weights: []
-    - serializedVersion: 2
-      name: Orc rider-Attack02_12
-      rect:
-        serializedVersion: 2
-        x: 673
-        y: 29
-        width: 20
-        height: 34
-      alignment: 0
-      pivot: {x: 0, y: 0}
-      border: {x: 0, y: 0, z: 0, w: 0}
-      customData: 
-      outline: []
-      physicsShape: []
-      tessellationDetail: -1
-      bones: []
-      spriteID: 63b85931bc37bd710800000000000000
-      internalID: 1719094998816099126
-      vertices: []
-      indices: 
-      edges: []
-      weights: []
     outline: []
     customData: 
     physicsShape: []
     bones: []
-    spriteID: 
+    spriteID: a75107854fe55224f8d21635e25f1b85
     internalID: 0
     vertices: []
     indices: 
@@ -437,9 +349,6 @@ TextureImporter:
     nameFileIdTable:
       Orc rider-Attack02_0: -4805291195941671494
       Orc rider-Attack02_1: -898491107783281163
-      Orc rider-Attack02_10: -8545410276645888226
-      Orc rider-Attack02_11: 6831388209841190207
-      Orc rider-Attack02_12: 1719094998816099126
       Orc rider-Attack02_2: 2851206345125205499
       Orc rider-Attack02_3: 5773026405177017699
       Orc rider-Attack02_4: -3239606791055260546
@@ -447,7 +356,6 @@ TextureImporter:
       Orc rider-Attack02_6: 837233935513997722
       Orc rider-Attack02_7: 7414343374642175032
       Orc rider-Attack02_8: 6936758103650135619
-      Orc rider-Attack02_9: -2948846752720918047
   mipmapLimitGroupName: 
   pSDRemoveMatte: 0
   userData: 
diff --git a/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider/Orc rider-Attack03.png.meta b/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider/Orc rider-Attack03.png.meta
index c070060d..da578eed 100644
--- a/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider/Orc rider-Attack03.png.meta	
+++ b/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider/Orc rider-Attack03.png.meta	
@@ -67,7 +67,7 @@ TextureImporter:
   maxTextureSize: 2048
   textureSettings:
     serializedVersion: 2
-    filterMode: 1
+    filterMode: 0
     aniso: 1
     mipBias: 0
     wrapU: 1
@@ -133,10 +133,10 @@ TextureImporter:
       name: Orc rider-Attack03_0
       rect:
         serializedVersion: 2
-        x: 36
-        y: 42
-        width: 34
-        height: 30
+        x: 0
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -155,10 +155,10 @@ TextureImporter:
       name: Orc rider-Attack03_1
       rect:
         serializedVersion: 2
-        x: 136
-        y: 42
-        width: 33
-        height: 31
+        x: 100
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -177,10 +177,10 @@ TextureImporter:
       name: Orc rider-Attack03_2
       rect:
         serializedVersion: 2
-        x: 236
-        y: 42
-        width: 33
-        height: 35
+        x: 200
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -199,10 +199,10 @@ TextureImporter:
       name: Orc rider-Attack03_3
       rect:
         serializedVersion: 2
-        x: 336
-        y: 42
-        width: 34
-        height: 42
+        x: 300
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -221,10 +221,10 @@ TextureImporter:
       name: Orc rider-Attack03_4
       rect:
         serializedVersion: 2
-        x: 436
-        y: 42
-        width: 40
-        height: 40
+        x: 400
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -243,10 +243,10 @@ TextureImporter:
       name: Orc rider-Attack03_5
       rect:
         serializedVersion: 2
-        x: 536
-        y: 42
-        width: 59
-        height: 29
+        x: 500
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -265,10 +265,10 @@ TextureImporter:
       name: Orc rider-Attack03_6
       rect:
         serializedVersion: 2
-        x: 608
-        y: 42
-        width: 87
-        height: 29
+        x: 600
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -287,10 +287,10 @@ TextureImporter:
       name: Orc rider-Attack03_7
       rect:
         serializedVersion: 2
-        x: 708
-        y: 42
-        width: 61
-        height: 29
+        x: 700
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -309,10 +309,10 @@ TextureImporter:
       name: Orc rider-Attack03_8
       rect:
         serializedVersion: 2
-        x: 811
-        y: 42
-        width: 60
-        height: 29
+        x: 800
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -331,10 +331,10 @@ TextureImporter:
       name: Orc rider-Attack03_9
       rect:
         serializedVersion: 2
-        x: 936
-        y: 42
-        width: 59
-        height: 29
+        x: 900
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -353,10 +353,10 @@ TextureImporter:
       name: Orc rider-Attack03_10
       rect:
         serializedVersion: 2
-        x: 1036
-        y: 42
-        width: 33
-        height: 29
+        x: 1000
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -375,7 +375,7 @@ TextureImporter:
     customData: 
     physicsShape: []
     bones: []
-    spriteID: 
+    spriteID: 28be778561aef1745bf990c6ef3224ca
     internalID: 0
     vertices: []
     indices: 
diff --git a/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider/Orc rider-Block.png.meta b/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider/Orc rider-Block.png.meta
index 91c75456..f044e6b7 100644
--- a/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider/Orc rider-Block.png.meta	
+++ b/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider/Orc rider-Block.png.meta	
@@ -46,7 +46,7 @@ TextureImporter:
   maxTextureSize: 2048
   textureSettings:
     serializedVersion: 2
-    filterMode: 1
+    filterMode: 0
     aniso: 1
     mipBias: 0
     wrapU: 1
@@ -112,10 +112,10 @@ TextureImporter:
       name: Orc rider-Block_0
       rect:
         serializedVersion: 2
-        x: 36
-        y: 42
-        width: 34
-        height: 30
+        x: 0
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -134,10 +134,10 @@ TextureImporter:
       name: Orc rider-Block_1
       rect:
         serializedVersion: 2
-        x: 136
-        y: 42
-        width: 33
-        height: 29
+        x: 100
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -156,10 +156,10 @@ TextureImporter:
       name: Orc rider-Block_2
       rect:
         serializedVersion: 2
-        x: 236
-        y: 42
-        width: 33
-        height: 29
+        x: 200
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -178,10 +178,10 @@ TextureImporter:
       name: Orc rider-Block_3
       rect:
         serializedVersion: 2
-        x: 336
-        y: 42
-        width: 34
-        height: 29
+        x: 300
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -200,7 +200,7 @@ TextureImporter:
     customData: 
     physicsShape: []
     bones: []
-    spriteID: 
+    spriteID: 1d8aca02e6574724785f56db0f6dac86
     internalID: 0
     vertices: []
     indices: 
diff --git a/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider/Orc rider-Death.png.meta b/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider/Orc rider-Death.png.meta
index 8931415f..59406b8a 100644
--- a/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider/Orc rider-Death.png.meta	
+++ b/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider/Orc rider-Death.png.meta	
@@ -46,7 +46,7 @@ TextureImporter:
   maxTextureSize: 2048
   textureSettings:
     serializedVersion: 2
-    filterMode: 1
+    filterMode: 0
     aniso: 1
     mipBias: 0
     wrapU: 1
@@ -112,10 +112,10 @@ TextureImporter:
       name: Orc rider-Death_0
       rect:
         serializedVersion: 2
-        x: 36
-        y: 42
-        width: 34
-        height: 30
+        x: 0
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -134,10 +134,10 @@ TextureImporter:
       name: Orc rider-Death_1
       rect:
         serializedVersion: 2
-        x: 136
-        y: 42
-        width: 33
-        height: 29
+        x: 100
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -156,10 +156,10 @@ TextureImporter:
       name: Orc rider-Death_2
       rect:
         serializedVersion: 2
-        x: 232
-        y: 41
-        width: 45
-        height: 20
+        x: 200
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -178,10 +178,10 @@ TextureImporter:
       name: Orc rider-Death_3
       rect:
         serializedVersion: 2
-        x: 327
-        y: 41
-        width: 54
-        height: 15
+        x: 300
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -200,7 +200,7 @@ TextureImporter:
     customData: 
     physicsShape: []
     bones: []
-    spriteID: 
+    spriteID: b393ab684bb03e14a89c5d63cd9af1cb
     internalID: 0
     vertices: []
     indices: 
diff --git a/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider/Orc rider-Hurt.png.meta b/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider/Orc rider-Hurt.png.meta
index 4d9130fb..d462d0f3 100644
--- a/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider/Orc rider-Hurt.png.meta	
+++ b/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider/Orc rider-Hurt.png.meta	
@@ -46,7 +46,7 @@ TextureImporter:
   maxTextureSize: 2048
   textureSettings:
     serializedVersion: 2
-    filterMode: 1
+    filterMode: 0
     aniso: 1
     mipBias: 0
     wrapU: 1
@@ -112,10 +112,10 @@ TextureImporter:
       name: Orc rider-Hurt_0
       rect:
         serializedVersion: 2
-        x: 36
-        y: 42
-        width: 34
-        height: 29
+        x: 0
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -134,10 +134,10 @@ TextureImporter:
       name: Orc rider-Hurt_1
       rect:
         serializedVersion: 2
-        x: 136
-        y: 42
-        width: 34
-        height: 29
+        x: 100
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -156,10 +156,10 @@ TextureImporter:
       name: Orc rider-Hurt_2
       rect:
         serializedVersion: 2
-        x: 236
-        y: 42
-        width: 33
-        height: 29
+        x: 200
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -178,10 +178,10 @@ TextureImporter:
       name: Orc rider-Hurt_3
       rect:
         serializedVersion: 2
-        x: 336
-        y: 42
-        width: 33
-        height: 29
+        x: 300
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -200,7 +200,7 @@ TextureImporter:
     customData: 
     physicsShape: []
     bones: []
-    spriteID: 
+    spriteID: c6ddd0215a6fa35468f516297f475062
     internalID: 0
     vertices: []
     indices: 
diff --git a/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider/Orc rider-Idle.png.meta b/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider/Orc rider-Idle.png.meta
index c962f495..543c0e7d 100644
--- a/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider/Orc rider-Idle.png.meta	
+++ b/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider/Orc rider-Idle.png.meta	
@@ -52,7 +52,7 @@ TextureImporter:
   maxTextureSize: 2048
   textureSettings:
     serializedVersion: 2
-    filterMode: 1
+    filterMode: 0
     aniso: 1
     mipBias: 0
     wrapU: 1
@@ -118,10 +118,10 @@ TextureImporter:
       name: Orc rider-Idle_0
       rect:
         serializedVersion: 2
-        x: 36
-        y: 42
-        width: 34
-        height: 30
+        x: 0
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -140,10 +140,10 @@ TextureImporter:
       name: Orc rider-Idle_1
       rect:
         serializedVersion: 2
-        x: 136
-        y: 42
-        width: 34
-        height: 30
+        x: 100
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -162,10 +162,10 @@ TextureImporter:
       name: Orc rider-Idle_2
       rect:
         serializedVersion: 2
-        x: 236
-        y: 42
-        width: 33
-        height: 31
+        x: 200
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -184,10 +184,10 @@ TextureImporter:
       name: Orc rider-Idle_3
       rect:
         serializedVersion: 2
-        x: 336
-        y: 42
-        width: 33
-        height: 31
+        x: 300
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -206,10 +206,10 @@ TextureImporter:
       name: Orc rider-Idle_4
       rect:
         serializedVersion: 2
-        x: 436
-        y: 42
-        width: 33
-        height: 30
+        x: 400
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -228,10 +228,10 @@ TextureImporter:
       name: Orc rider-Idle_5
       rect:
         serializedVersion: 2
-        x: 536
-        y: 42
-        width: 33
-        height: 30
+        x: 500
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -250,7 +250,7 @@ TextureImporter:
     customData: 
     physicsShape: []
     bones: []
-    spriteID: 
+    spriteID: 8731766374f68454da843a72a9ced7dc
     internalID: 0
     vertices: []
     indices: 
diff --git a/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider/Orc rider-Walk.png.meta b/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider/Orc rider-Walk.png.meta
index 3572c725..94fd72a6 100644
--- a/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider/Orc rider-Walk.png.meta	
+++ b/Assets/Sprites/Tiny RPG Character Asset Pack v1.03 -Full 20 Characters/Characters(100x100)/Orc rider/Orc rider/Orc rider-Walk.png.meta	
@@ -58,7 +58,7 @@ TextureImporter:
   maxTextureSize: 2048
   textureSettings:
     serializedVersion: 2
-    filterMode: 1
+    filterMode: 0
     aniso: 1
     mipBias: 0
     wrapU: 1
@@ -124,10 +124,10 @@ TextureImporter:
       name: Orc rider-Walk_0
       rect:
         serializedVersion: 2
-        x: 36
-        y: 42
-        width: 34
-        height: 30
+        x: 0
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -146,10 +146,10 @@ TextureImporter:
       name: Orc rider-Walk_1
       rect:
         serializedVersion: 2
-        x: 136
-        y: 42
-        width: 33
-        height: 31
+        x: 100
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -168,10 +168,10 @@ TextureImporter:
       name: Orc rider-Walk_2
       rect:
         serializedVersion: 2
-        x: 236
-        y: 42
-        width: 33
-        height: 30
+        x: 200
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -190,10 +190,10 @@ TextureImporter:
       name: Orc rider-Walk_3
       rect:
         serializedVersion: 2
-        x: 336
-        y: 42
-        width: 33
-        height: 29
+        x: 300
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -212,10 +212,10 @@ TextureImporter:
       name: Orc rider-Walk_4
       rect:
         serializedVersion: 2
-        x: 436
-        y: 42
-        width: 33
-        height: 30
+        x: 400
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -234,10 +234,10 @@ TextureImporter:
       name: Orc rider-Walk_5
       rect:
         serializedVersion: 2
-        x: 536
-        y: 42
-        width: 33
-        height: 30
+        x: 500
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -256,10 +256,10 @@ TextureImporter:
       name: Orc rider-Walk_6
       rect:
         serializedVersion: 2
-        x: 636
-        y: 42
-        width: 34
-        height: 29
+        x: 600
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -278,10 +278,10 @@ TextureImporter:
       name: Orc rider-Walk_7
       rect:
         serializedVersion: 2
-        x: 736
-        y: 42
-        width: 35
-        height: 29
+        x: 700
+        y: 0
+        width: 100
+        height: 100
       alignment: 0
       pivot: {x: 0, y: 0}
       border: {x: 0, y: 0, z: 0, w: 0}
@@ -300,7 +300,7 @@ TextureImporter:
     customData: 
     physicsShape: []
     bones: []
-    spriteID: 
+    spriteID: 1b23379c5c7111f4fa1c9feb8b7849ec
     internalID: 0
     vertices: []
     indices: 
